packages/apps/explorer/src/components/select-network/ConfigNetwork.tsx

```diff
@@ -0,0 +1,279 @@
+                            ? 'primary'
+                            : 'transparent'
+                      }
+                      onPress={() => validateNetwork()}
+                    >
+                      Validate
+                    </Button>
+                  }
+                ></TextField>
+
+                {network.graphUrlIsValid && (
+                  <>
+                    <TextField
+                      maxLength={15}
+                      label="Name"
+                      name="label"
+                      value={network?.label}
+                      onChange={(e) => {
+                        setNetwork((v) => ({ ...v, label: e.target.value }));
+                      }}
+                      autoFocus
+                      isRequired
+                    ></TextField>
+                    <TextField
+                      maxLength={50}
+                      label="NetworkId"
+                      value={network?.networkId}
+                      onChange={(e) => {
+                        setNetwork((v) => ({
+                          ...v,
+                          networkId: e.target.value,
+                        }));
+                      }}
+                      name="networkId"
+                      isRequired
+                    ></TextField>
+                    <TextField
+                      maxLength={15}
+                      label="Slug"
+                      name="slug"
+                      value={network?.slug}
+                      onChange={(e) => {
+                        setNetwork((v) => ({ ...v, slug: e.target.value }));
+                      }}
+                      isRequired
+                    ></TextField>
+                  </>
+                )}
+
+                {formError && formError.length > 0 && (
+                  <Stack flexDirection="column">
+                    {(formError as string[]).map((e) => (
+                      <Text key={e} color="emphasize">
+                        {e}
+                      </Text>
+                    ))}
+                  </Stack>
+                )}
+              </Stack>
+            </CardContentBlock>
+            <CardFooter>
+              {!isDefaultNetwork(network) && !network.isNew && (
+                <Button onPress={handleRemove} variant="negative">
+                  Remove
+                </Button>
+              )}
+              <Stack flex={1} />
+
+              {!isDefaultNetwork(network) && (
+                <>
+                  <Button
+                    onPress={() => {
+                      handleOpen(false);
+                    }}
+                    variant="outlined"
+                  >
+                    Cancel
+                  </Button>
+                  <Button type="submit">
+                    {!network.isNew ? 'Update network' : 'Create Network'}
+                  </Button>
+                </>
+              )}
```

Comment:
Are you trying to have buttons with space between? If so, there should be a prop on the CardFooter that allows for this (separated)